app/views/trends/_extraction_warning.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if User.current.admin? %>
+  <%= alert :class => 'alert-warning', :header => _("This funcionality is deprecated"),
+            :text => ("and will be removed in version 2.2. "\
+                       "If you wish continue using it, please install a plugin Foreman Statistics. "\
+                       "For instructions follow " +
```

Comment:
Rather than this, should `the Foreman Statistics plugin` be a link to the actual manual page?

Reply:
Yeah, I guess it would work better :+1: